app/containers/HomePage/index.js

```diff
@@ -22,9 +22,15 @@ class HomePage extends React.Component {
     this.onChangeRoute = this.onChangeRoute.bind(this);
     this.changeRouteToReadme = this.changeRouteToReadme.bind(this);
   }
+
+  shouldComponentUpdate(nextState, prevState) {
```

Comment:
https://facebook.github.io/react/docs/component-specs.html#updating-shouldcomponentupdate gets passed in nextProps and nextState

comparing them takes actually more effort. I recommend to use this https://github.com/gaearon/react-pure-render